doc/conf.py

```diff
@@ -33,12 +33,13 @@
 nitpicky = True
 show_warning_types = True
 
-html_theme = 'sphinx13'
-html_theme_path = ['_themes']
-html_css_files = [
-    # 'basic.css',  # included through inheritance from the basic theme
-    'sphinx13.css',
-]
+# html_theme = 'sphinx13'
+html_theme = 'basic'
+html_theme_path = ['../sphinx/themes']
+# html_css_files = [
+#     # 'basic.css',  # included through inheritance from the basic theme
+#     'sphinx13.css',
+# ]
```

Comment:
This is just my WIP code to apply the changes to upstream Sphinx docs. None of this should be merged.

Reply:
Removed